lib/arrow_web/controllers/disruption_controller/filters.ex

```diff
@@ -22,11 +22,12 @@ defmodule ArrowWeb.DisruptionController.Filters do
 
   @type t :: %__MODULE__{
           routes: MapSet.t(String.t()),
+          only_approved: boolean(),
```

Comment:
suggestion: Elixir-y booleans end with `?`
```suggestion
          only_approved?: boolean(),
```
(not going to re-tag it everywhere)

Reply:
Should I also update `Table.include_past`?

Reply:
In general yes, but I'll leave it to you as to whether that's out-of-scope here.